src/Migrate/custom/New-AzMigrateServerReplication.ps1

```diff
@@ -227,6 +228,38 @@ function New-AzMigrateServerReplication {
 
             $null = $PSBoundParameters.Remove('MachineId')
             $null = $PSBoundParameters.Remove('InputObject')
+
+            # Validate some input.
+            if ($LicenseType -eq "NotSpecified"){
```

Comment:
please use the following style of “if else"
```powershell
if (something) {
}
elseif (something) 
```
Please take care of the space between "if" and "(" or ")" and "{".

Reply:
I  have removed the if-else blocks, and used ValidateSet.

Reply:
I have formatted the other custom ps1 files as well.